### PR TITLE
Completed prep-javascript-dom-manipulation.

### PR DESCRIPTION
**Pull request by stefancarrera**
java7

### CODE REVIEW

File: prep-javascript-dom-manipulation/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <p class="second-paragraph">
+
+    </p>
+    <p class="third-paragraph">
+
+    </p>
+  </div>
+  <script>
+    // Your code here
+  </script>
```
Comment (thebearingedge) [inline prep-javascript-dom-manipulation/index.html]: ??? What happened to the JavaScript? 😱

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Somebody stole your JavaScript!

**Comment by stefancarrera** [comment]
java7-1

**Review by thebearingedge — APPROVED** [review]
Approved!

**Pull request merged**